src/core/store/modules/lines/lines.selectors.ts

```diff
@@ -198,6 +200,14 @@ const selectPositions = createSelector([selectSelectedLine], (line) => {
   return positions;
 });
 
+// TODO: this should return the equivalent of CreditLineState.user.borrowerPositions
```

Comment:
You dont need this you already have a selector that does exactly this and is called the same thing. Just delete